container/containerd/client.go

```diff
@@ -130,6 +146,23 @@ func (c *client) TaskPid(ctx context.Context, id string) (uint32, error) {
 	return response.Process.Pid, nil
 }
 
+func (c *client) RootfsDir(ctx context.Context) (string, error) {
+	r, err := c.runtimeService.Status(ctx, &runtimeapi.StatusRequest{Verbose: true})
+	if err != nil {
+		return "", err
+	}
+	configStr := r.Info["config"]
+	config := make(map[string]interface{})
+	if err := json.Unmarshal([]byte(configStr), &config); err != nil {
```

Comment:
Could you unmarshal config into a struct? It will be easier to grasp what values are interesting.

Reply:
The original definition of config is [here](https://github.com/containerd/containerd/blob/c17839bdc7bf8699badf796078afc91fd54de6ac/internal/cri/config/config.go#L411-L424)

I extracted the struct in config.go to container/containerd/config